### PR TITLE
Sales rep map view add customer category selectors

### PR DESCRIPTION
**Pull request by soft-Dev**

### CODE REVIEW

File: roles/sales-rep/map-view/js/customer.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
-define([], function () {
- 
-    function Select (bezl, custNum) {
+define(["./map.js"], function (map) {
+
+    function Select(bezl, custNum) {
+        console.log(bezl.vars.markers[custNum].data);
```
Comment (bitingmidge) [inline roles/sales-rep/map-view/js/customer.js]: @soft-Dev Do you want this console.log() call here or was this left over debugging?

**Comment by bitingmidge** [comment]
@soft-Dev Looks like this requires a GetFilterCategories.sql file... is that included in the pull request?

**Comment by soft-Dev** [comment]
@bitingmidge Added missing SQL file and removed logging line.

**Comment by bitingmidge** [comment]
Closes #195

**Pull request merged**